ietf/meeting/models.py

```diff
@@ -1167,7 +1167,7 @@ def can_manage_materials(self, user):
         return can_manage_materials(user,self.group)
 
     def is_material_submission_cutoff(self):
-        return date_today(self.meeting.tz()) > self.meeting.get_submission_correction_date()
+        return date_today(datetime.timezone.utc) > self.meeting.get_submission_correction_date()
```

Comment:
good find. Makes no sense that this was tied to meeting timezone.

Reply:
Thanks (though I'm probably responsible for tying it there to begin with....)